easybuild/easyconfigs/c/cuDNN/cuDNN-7.0.5.15-fosscuda-2017b.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# OriginalAuthor:    Stephane Thiell <sthiell@stanford.edu>
+# Author:    Ake Sandgren <ake.sandgren@hpc2n.umu.se>
+##
+easyblock = 'Tarball'
+
+# The full version of the library can be found using
+# strings -a cuda/lib64/libcudnn_static.a | grep cudnn_version_
+# Download and rename.
+name = 'cuDNN'
+version = '7.0.5.15'
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for
+    deep neural networks."""
+
+# fosscuda 2017b uses CUDA 9.0
+toolchain = {'name': 'fosscuda', 'version': '2017b'}
+
+# Nvidia developer registration required.
+# Download link:
+# https://developer.nvidia.com/compute/machine-learning/cudnn/secure/v7.0.5/prod/9.1_20171129/cudnn-9.0-linux-x64-v7
+# Complete version number is taken from the corresponding .deb files from the same URL base
```

Comment:
We should perhaps change this comment to match what's written above regarding "strings -a ..."

Reply:
And the same goes for the intelcuda version